src/main/java/com/redhat/foreman/ForemanSharedNodeCloud.java

```diff
@@ -472,7 +472,7 @@ public FormValidation doTestConnection(@QueryParameter("url") String url,
                     return FormValidation.error(Messages.Error() + ": " + e);
                 }
             }
-            return FormValidation.error(Messages.InvalidURI());
+            return FormValidation.error("Invalid Foreman API Url");
```

Comment:
My bad of course, I thought the content of message comes from the Jenkins core, will fix it.

Anyway we should move to the catalog all messages uses for WebGUI feedback.